Documentation/subcommands/run.md

```diff
@@ -25,6 +25,11 @@ rkt will automatically [fetch](fetch.md) them if they're not present in the loca
 # rkt --insecure-options=image run docker://quay.io/coreos/etcd:v2.0.0
 ```
 
+### Run multiple applications in the same pod
```

Comment:
two nits:
1) could you please change ### -> ## ? I think it should be a higher level section
2) could you add a preamble? it can be brief and simple, e.g. "Multiple applications can be run in a pod by passing multiple images to the run command"